docs/endpoints/openrtb2/auction.md

```diff
@@ -184,6 +185,8 @@ For backwards compatibility the following strings will also be allowed as price
 
 One of "includewinners" or "includebidderkeys" must be true (both default to true if unset). If both were false, then no targeting keys would be set, which is better configured by omitting targeting altogether.
 
+The parameter "includeformat" is for multiformat support. It will add the key `hb_format` and/or `hb_format_{bidderName}` as per "includewinners" and "includebidderkeys" above.
```

Comment:
What do you think about modifying this a little bit to say:
```
The parameter "includeformat" indicates the type of the bid (banner, video, etc) for multiformat requests. It will add the key ....
```